docker-compose.dev.yml

```
@@ -13,8 +13,25 @@ services:
       - db
       - redis
+      - llm_service
       - minio
-#      - ollama
+
+  llm_service:
+    build:
+      context: ./llm_service
+      dockerfile: ../docker/Dockerfile.llm
+    volumes:
+      - ./llm_service:/app 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)